### DISCUSSION TITLE
Batocera Lite/Koriki Pre-Alpha for RG35XX Plus - 20231217

### DISCUSSION THREAD

**Discussion by acmeplus** [rg35xx-cfw/rg35xx-cfw.github.io#66] [Announcements]
Use this  discussion to provide feedback and comment on bugs or features on this release. As mentioned on the release page there are still many things missing in this release, and things may not work as expected.

[Release Page](https://github.com/rg35xx-cfw/rg35xx-cfw.github.io/releases/tag/rg35xx_plus_batocera_koriki_lite_pre-alpha_20231217)

**Comment by se1000** [comment]
Just got it fired up, so far:
- /share/system/logs folder doesn't get created and causes system to throw error 256
- /share/roms directories don't get auto generated (at least on exFat)
- seems to be some weird interpolation, looks like some kind of bi-linear filtering even when the setting is turned off (SNES)
- audio isn't 100% smooth

**Reply by acmeplus** [reply]
> Just got it fired up, so far:
> 
> - /share/system/logs folder doesn't get created and causes system to throw error 256

Interesting. I thought I fixed that bug before the release. Will investigate. 

For those having the error 256 in EmulationStation when trying to launch a rom, insert the sdcard on your computer and create a folder system/logs on the last partition (SHARE).

**Reply by se1000** [reply]
FYI, I deleted the normal /share partition and re-created my own exFat one, maybe related.

Threaded video seems to help the ever so slight audio crackle (Snes9x)

**Reply by acmeplus** [reply]
Not sure if I included supafaust there. That’s a good candidate to try.

**Reply by acmeplus** [reply]
@se1000 I found the issue there, or at least something that may point to the actual problem. The clock speed has been incorrectly set to the default RG35XX model speed. The S02overclock init script needs to be updated.

In the meantime you can change the speed via ssh and typing:
``echo 1512000 > /sys/devices/system/cpu/cpu0/cpufreq/scaling_max_freq``

**Reply by se1000** [reply]
Just tried the fix and it definitely seems to be it

**Comment by Hypervisor64** [comment]
Been dark for a while. sure you are working on this like crazy. just wondering when there might be any kind of updates? i'm super stoked about it .

**Reply by acmeplus** [reply]
I've been busy trying to fix pending issues for the original RG35XX. I'm actually getting slowly back to the plus, and hope to have an update at some point this week. Hopefully things like brightness control, audio pop sounds and other issues should be solved. Still a lot to do to get this to perform well.

**Reply by Hypervisor64** [reply]
well you are doing great work and every one appreciates you . People are really excited for this one .  Honestly the 2 things id like to see most that would make me full time batocera on the plus is just retro achievements and over air updates. past that its damn near there already  in terms of just using batocera  as the main OS all the time  from a laymen's perspective.

**Comment by johnsledget** [comment]
Test result:
Ports - game Wolfenstein 3D, does not open. it is possible the sh file is not correctly implemented.
Atomiswave - Guilty Gear X ver. 1.5. It does not open and stay black screen. Warning bang "Please upgrade to MAME romsets or expect issues". I force quit Menu+Start button.
SG-1000 - 007 James Bond. No sound. Button map works and the game ran good.
Master system - Renegade (Europe). No sound. Button map works and the game ran good.
Mega drive - Aladdin (USA). No sound. Button map works and the game ran good.
Game gear - The Lion King (USA). No sound. Button map works and the game ran good.
Mega CD - Vay. No sound. Button map works and the game ran good.
Sega saturn - Xmen children of atom. Warning: the system has missing bios files pop up. No sound. Button map works and the game ran good.
Dreamcast - Crazy Taxi (USA). Stuck when trying openning the game. Black screen hang up. I force quit Menu+Start button to exit.
Sega Naomi - Azumanga. It does not open and stay black screen. Warning bang "Please upgrade to MAME romsets or expect issues". I force quit Menu+Start button.
Sharp x68000 - Super Mario Bros. Game open/run fine but does have button map.
ZX Spectrum - Out run. It runs but I don't know how to play. It let me try to write something using a virtual keyboard.
Neogeo - Art of fighting. No sound. Button map works and the game ran good.
Neogeo CD - Fatal fury fight of destiny. Warning: the system has missing bios files pop up. No sound. Button map works and the game ran good.
Neogeo pocket - Neo cherry master. No sound. Button map works and the game ran good.
Neogeo pocket color - Baseball stars color. No sound. Button map works and the game ran good.
Playstation - bloody roar II (usa).  Warning: the system has missing bios files pop up. No sound. Button map works and the game ran good.

PSP - Grandtheftauto liberty city stories.  Audio works. Button map works and the game ran good.

Supervision - Q-Bert 1983. Does not open.
Amstrad CPC - Batman the movie. No sound. No Button map and the game ran good.
Amstrad GX4000 - Robocop 2. Does not open.
Arcade Mame - Video Eight ball. No sound. Button map works and the game ran good.
FB Neo - Frogger. No sound. Button map works and the game ran good.
Atari 2600 - 2005 minigame multicart. No sound. Button map works and the game ran good.
Atari 8bits - Basketball. No sound. Button map works and the game ran good.
Atari 5200 - A.E. No sound. Button map works and the game ran good.
Atari ST - Adventure (USA). Warning: the system has missing bios files pop up. Does not open.
Atari 7800 - Ace of aces. No sound. Button map works and the game ran good.
Lynx - ninja gaiden 1990. Does not open
Atari jaguar - Aliens versus predator. Warning the emulator/core currently set does not support this games file format. Does not open
Wonderswan - Crazy climber. No audio. Button mapped okay. Game run okay.
Wonderswancolor - Digital Monster. No audio. Button mapped okay. Game run okay.
Colecovision - Antartic adventure. No audio. Button mapped okay. Game run okay.
Commodore 64 - Armalyte. No audio. Button mapped okay. Game run okay.
Intellivision - 4-tris.No audio. Button mapped okay. Game run okay.
Vectrex - Armor attack.No audio. Button mapped okay. Game run okay.
Msdos - xentar. It has interface but i dont know how to make it play.
Msx -1942. Warning system has missing bios files. No audio. Button mapped okay. Game run okay.
Msx2 - 1942. No audio. Button mapped okay. Game run okay.
Pc Engine - columns. No audio. Button mapped okay. Game run okay.
Pc EngineCD - Double dragon ii. No audio. Button mapped okay. Game run okay.
Supergrafx - bomberman 93.No audio. Button mapped okay. Game run okay.
Pcfx -toNari.No audio. Button mapped okay. Game run okay.
Lcd games - bomb fight.No audio. Button mapped okay. Game run okay.
Nes - perfect bowling.No audio. Button mapped okay. Game run okay.
Gameboy - super mario land.No audio. Button mapped okay. Game run okay.
Snes - american battle dome.No audio. Button mapped okay. Game run okay.
N64 - 1080 snowboarding. Has audio and plays good
Gbc - 1942.No audio. Button mapped okay. Game run okay.
Gba - advance wars. Warning system missing bios files.No audio. Button mapped okay. Game run okay.
Pokemonmini - pokemon shock tetrisNo audio. Button mapped okay. Game run okay.
Nds - meteos. Warning system missing bios files. Does not open
3DO - optidoom.No audio. Button mapped okay. Game run okay.
OpenBor - armored warrior. Has audio and works good.

**Reply by acmeplus** [reply]
Check the new release, the audio issues should have now be resolved.

**Comment by ericarrache** [comment]
Nintendo DS is not working even though I put the right bios files. Tried multiple games

**Comment by acmeplus** [comment]
New update released: https://github.com/rg35xx-cfw/rg35xx-cfw.github.io/releases/tag/rg35xx_plus_batocera_lite_koriki_alpha_20240116
